ext4/internal/compactext4/verify_linux_test.go

```diff
@@ -44,8 +47,14 @@ func llistxattr(path string, b []byte) (int, error) {
 }
 
 func lgetxattr(path string, name string, b []byte) (int, error) {
-	pathp := syscall.StringBytePtr(path)
-	namep := syscall.StringBytePtr(name)
+	pathp, err := syscall.BytePtrFromString(path)
+	if err != nil {
+		return 0, &os.PathError{Path: path, Op: "llistxattr", Err: err}
```

Comment:
Do we need to set error with `syscall.Errno(e)` as it is done elsewhere?

Reply:
I dont think so, since `BytePtrFromString` says it either returns `nil` or `syscall.EINVAL`.
Also, the return is already an `error` (and not an `Errno` or `uintptr`), so I would have to `error.(syscall.Errno)` just to then save it as an `error` field.

Reply:
Sounds good! Not sure why we even have to do the `syscall.Errno(e)` casting in other places. `Errno` is already of Error type.